docs-source/content/security/openshift.md

```diff
@@ -5,14 +5,111 @@ weight: 7
 description: "OpenShift information for the operator"
 ---
 
-#### OpenShift `anyuid` security context
-
-The Docker images that Oracle publishes default to the container user
-as `oracle`, which is UID `1000` and GID `1000`. When running the
-Oracle images or layered images that retain the default user as
-`oracle` with OpenShift, the `anyuid` security context constraint
-is required to ensure proper access to the file system within the
-Docker image. This means that the administrator must:
+#### Security requirements to run WebLogic in OpenShift
+
+WebLogic Server Docker images obtained from Oracle Container Registry have an `oracle` user with
+UID 1000 and it is in a group, which is also called `oracle` and has GID 1000.
```

Comment:
This is no longer true. Now the images have user `oracle` that is part of the `root` group. We did this precisely so that the images would work better with OpenShift. I'm going to keep reading, but my guess is that much of the rest of this content is affected by this change. Looping in @mriccell.

Reply:
updated as discussed